src/modules/header/styles.css

```diff
@@ -0,0 +1,246 @@
+  padding-bottom: var(--app-bar-padding-y, var(--component-padding-y));
+  margin: 0;
+  font-size: 1rem;
+  color: hsl(var(--component-color) / var(--component-medium-emphasis));
+  text-align: left;
+
+  /* Inherited from Material's Subtitle 1 */
+  text-transform: none;
+  letter-spacing: .01em;
+  border-radius: 0;
+
+  @media (--header-breakpoint) {
+    width: auto;
+  }
+
+  &::after {
+    position: absolute;
+    top: var(--app-bar-padding-y, var(--component-padding-y));
+    right: var(--app-bar-padding-x, var(--component-padding-x));
+    display: inline-block;
+    width: 1.5rem;
+    height: 1.5rem;
+    margin: -.25rem;
+    vertical-align: middle;
+    content: '';
+    background:
+      hsl(
+        var(--app-bar-color, var(--component-color)) /
+        var(--app-bar-disabled, var(--component-disabled))
+      );
+    transition: transform 50ms;
+    clip-path: path('M7.41,8.58L12,13.17L16.59,8.58L18,10L12,16L6,10L7.41,8.58Z');
+
+    @media (--header-breakpoint) {
+      position: static;
+      margin-left: .25rem;
+    }
+  }
+
+  &[aria-expanded='true'] {
+    color: var(--primary);
+
+    &::after {
+      transform: rotate(180deg);
+    }
+  }
+}
+
+.menu-list {
+  display: none;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+
+  &.expanded {
+    display: flex;
+  }
+
+  @media (--header-breakpoint) {
+    position: absolute;
+    top: 100%;
+    right: 0;
+    z-index: var(--header-zindex, 1000);
+    width: var(--header-menu-width);
+    background: hsl(var(--app-bar-background, var(--component-background)));
+    box-shadow: var(--elevation-04);
+  }
+}
+
+.group {
+  padding:
+    var(--app-bar-padding-y, var(--component-padding-y))
+    var(--app-bar-padding-x, var(--component-padding-x));
+
+  & .item {
+    margin-right: calc(-1 * var(--app-bar-padding-x, var(--component-padding-x)));
+    margin-left: calc(-1 * var(--app-bar-padding-x, var(--component-padding-x)));
+  }
+}
+
+.group-title {
+  margin: 0;
+  font-size: .875rem;
+  font-weight: 400;
+  color: hsl(var(--component-color) / var(--component-disabled));
+  letter-spacing: .01em;
+}
+
+.group-list {
+  padding: 0;
+  margin: 0;
+}
+
+.item {
+  display: block;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.item-link {
+  display: block;
+  padding: .5rem var(--app-bar-padding-x, var(--component-padding-x));
+  margin: 0;
+  line-height: 1;
+
+  &:hover {
+    text-decoration: none;
+    background-color: hsl(var(--app-bar-overlay, var(--component-overlay) / var(--component-overlay-hover)));
+  }
+
+  &:focus {
+    text-decoration: none;
+    background-color: hsl(var(--app-bar-overlay, var(--component-overlay) / var(--component-overlay-focus)));
+    outline: 0;
+  }
+}
+
+/* Special menus */
+.services-menu {
+  @media (--header-breakpoint) {
+    & .menu-list {
+      /* flex-direction: row; */
+      flex-wrap: wrap;
+      width: calc(3 * var(--header-menu-width));
```

Comment:
It may overflow. Maybe you should break it sooner?

![image](https://user-images.githubusercontent.com/12775440/98534524-e81a5780-2284-11eb-9edf-0846de7e339c.png)

Reply:
No, the menu is smaller than `768px`. The breakpoint is applied to the full width of the screen when the menu is visible only on part of it. It should be safe to leave it as is.